src/swagger-specs/destinations.yaml

```diff
@@ -34,81 +34,313 @@ servers:
+          For destinations, the source that you are connecting to is Experience Platform data and the target is the desired destination. 
+          The base connection configuration always includes a unique `connectionSpec ID`, which is necessary in order to successfully access Experience Platform data and authorize a connection to the target destination.
+  - name: Source connections
+    description: Source connections create and manage a connection to the internal source from where data is exported. For destination connections, the source connection is always the Experience Platform Profile Store. Source connections contain information like data source, data format, and the source connection ID needed to create a dataflow. A source connection instance is specific to a tenant and IMS Organization.
+  - name: Target connections
+    description: Target connections create and manage a destination connection to any location where exported data will land. Target connections contain information regarding data destination, data format, and the target connection ID needed to create a destination dataflow. A target connection instance is specific to a tenant and IMS Organization. <br><br> Before you create a target connection, you <i>must</i> create a base connection.
   - name: Dataflows
     description: Destination dataflows represent the data transfer between a source connection (Experience Platform) and a target connection (the destination you set up). This also includes information regarding the data export schedule, and the transformation specifications of the data being exported to your destination. A dataflow instance is specific to a tenant and Organization.
-  - name: Dataflow Runs
-    description: Dataflow runs represent an instance of a dataflow execution, which exports data to your destination. A run instance is specific to a tenant and IMS Organization.
+  - name: Dataflow runs
+    description: Dataflow runs represent an instance of a dataflow execution, which exports data to your destination. A dataflow run is specific to a dataflow and a segment for which the data export occurred. Furthermore, a run instance is specific to a tenant and Organization.
 
 paths:
-  /flows:
-    post:
+  /connections:
+    get:
       tags:
-        - Dataflows
-      summary: Create a destination dataflow
+       - Base connections
+      summary: Retrieve a list of destination base connections
       description: >
-       You can create a new destination dataflow by making a POST request to the `/flows` endpoint.
-      operationId: postFlow
+        You can retrieve a list of all destination base connections for your Organization by making a GET request to the `/connections` endpoint.
+        You can use query parameters to return only the base connections for a specific destination type. To do this, pass in the `connectionSpec.id` of the desired destination by using the `property` query parameter, as shown further below.
```

Comment:
Might want to add an extra blank line between these two, since for some reason when using the `>` syntax it needs two blank lines to actually render a linebreak. Currently these two lines appear as a single paragraph.